src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -186,6 +223,112 @@ private void HandleLocalResource(OpenDocumentationLinkEventArgs e)
             this.shouldLoadDefaultContent = false;
         }
 
+        private void WatchMdFile(string mdLink)
+        {
+            var watcher = packageManagerDoc.GetFileSystemWatcher(mdLink);
+            watcher.Changed += OnCurrentMdFileChanged;
+        }
+
+        private void UnsubscribeMdWatcher(Uri link)
+        {
+            if (link is null)
+                return;
+
+            var watcher = packageManagerDoc.GetFileSystemWatcher(link.OriginalString);
+            if (watcher is null)
+                return;
+
+            watcher.Changed -= OnCurrentMdFileChanged;
+        }
+
+        private void OnCurrentMdFileChanged(object sender, FileSystemEventArgs e)
+        {
+            if (e.FullPath != link.OriginalString)
+                return;
+            if (!(openDocumentationLinkEventArgs is OpenNodeAnnotationEventArgs))
+                return;
+
+            var nodeAnnotationArgs = openDocumentationLinkEventArgs as OpenNodeAnnotationEventArgs;
+            this.content = CreateNodeAnnotationContent(nodeAnnotationArgs);
+            OnContentChanged();
+        }
+
+        private string CreateNodeAnnotationContent(OpenNodeAnnotationEventArgs e)
+        {
+            var writer = new StringWriter();
+
+            var style = LoadContentFromResources("MarkdownStyling.html", false);
+            writer.WriteLine(style);
+
+            var header = CreateHeader(e);
+            writer.WriteLine(header);
+
+            var nodeInfo = CreateNodeInfo(e);
+            writer.WriteLine(nodeInfo);
+
+            var mdFilePath = packageManagerDoc.GetAnnotationDoc(e.NodeNamespace);
+            markdownHandler.GetAnnotationFromMd(ref writer, mdFilePath);
```

Comment:
as per discussion, we need to run the script tag remover on user content (MD) before adding our own for syntax highlighting
the below/above don't currently do that to the user content, they just specifically don't string JS for our portions.